teuthology/task/kernel.py

```diff
@@ -669,6 +669,7 @@ def enable_disable_kdb(ctx, config):
             except run.CommandFailedError:
                 log.warn('Kernel does not support kdb')
 
+
```

Comment:
this is extra

Reply:
would you prefer that I move it into a separate commit? In general, the Ceph dev/reviewer community tolerates small whitespace cleanups inside commits.

Reply:
I would agree if it is related some how to the method/function you're changing, thus I would prefer to do not mix and give it another commit. So, I would prefer you move it, however, I will not block this PR because of this.